### ISSUE TITLE
[Docs] Instructions for obtaining a `luisAuthoringKey`

### ISSUE THREAD

**Issue by losandes** [microsoft/botframework-solutions#2496]
/tutorials/typescript/create-assistant/1_intro/
/tutorials/typescript/create-assistant/4_provision_your_azure_resources/

Hi :wave:,

*tl;dr*: A link to documentation that helps users navigate the complexity of obtaining a `luisAuthoringKey` would be useful in this documentation.

---

In the instructions for provisioning azure resources (4_provision...), one of the required arguments to the deployment scripts is the `luisAuthoringKey`. This prerequisite is mentioned on step 1 (1_intro), but no links are provided.

I think this would be helpful because it appears that the process is a moving target. _Sign up_ for LUIS is presently broken (at least for me - I opened an issue for that too). I happened to have a personal account for LUIS from a couple years ago, and when I signed in with that, I was prompted to migrate my app to Azure. 

🤔 Maybe that's why I couldn't sign up. _types LUIS into Azure marketplace search... nothing_

Since I can't sign up at luis.ai, and I can't find LUIS in the Azure marketplace, it's not clear to me how new users would obtain an authoring key and be able to complete this demo.

On a side note - it feels a bit like a chicken and egg scenario. If LUIS is/will be an Azure resource, is it possible for the deployment scripts provision it and obtain the authoring key without requiring the prerequisite?

**Comment by lauren-mills** [comment]
Hi @losandes, the change to provision a LUIS authoring key in Azure (rather than only through the luis.ai portal) is a recent one. We will work with the LUIS team to update our documentation (and potentially the ARM templates, if possible) in the coming weeks. 

In the meantime, it looks like you can provision an authoring key in the Azure portal under the "Language Understanding" resource:

![image](https://user-images.githubusercontent.com/27976702/66510076-33ae7080-ea89-11e9-8149-3fa8245b47fd.png)

**Comment by losandes** [comment]
Thanks @lauren-mills. It looks like this demo doesn't work anymore. I get the following error on line 256 of deployment/scripts/deploy_cognitive_models.ps1 regardless of which pricing tier I choose.

```
The apps - Get applications list Operation under LUIS Programmatic APIs v2.0 API is not supported with the current subscription key and pricing tier
```

Of course that's being thrown by the dispatch package, so the issue probably doesn't belong on this repository. However that message followed by this error message that might not be true anymore:

```
Please assign your subscription key manually in the LUIS portal
```

**Comment by darrenj** [comment]
Thanks - updated with latest deployment scripts
